utils/test/RunningTests.md

```diff
@@ -0,0 +1,67 @@
+#Running tests for solutions-geoprocessing-toolbox
+
+##Sections
+* [Introduction](#introduction)
+* [Requirements](#requirements)
+	* [For ArcGIS Pro](#for-arcgis-pro)
+	* [For ArcGIS Desktop](#for-arcgis-desktop)
+* [Running the tests](#running-the-tests)
+* [Log files](#log-files)
+* [Reporting errors as Issues](#reporting-errors-as-issues)
+
+##Introduction
+The unittests in this section of the repository are designed to test the geoprocessing tools contained within to make sure they are running consistently under different conditions. Automated testing, unittests are one type, help reduce the amount of time necessary to find defects or confirm their success. An automated test can run all of the tools in a significantly shorter amount of time than several people testing the tools manually. This should not be taken as an opportunity to skip other types of testing (ad-hoc, defect verification, template testing, system testing, etc.).
+
+##Requirements
+###For ArcGIS Pro
+* ArcGIS Pro 1.1+
+* Python 3.4+
```

Comment:
Do you think you need to clarify that this is the separate/additional "ArcGISProPython34" installer (I know this still confuses me, if/when I need to install this - in this case I believe you do need this installer - to get py.exe and python3.exe)